src/js/cagov/sourcecode.js

```diff
@@ -21,7 +21,7 @@ var entityMap = {
  }
  
  function copyCode(btnElem) {
-  var codeblock = btnElem.parentElement.querySelector("pre code, textarea.sourcecode");
+  var codeblock = btnElem.previousSibling;
```

Comment:
Would it be better to use previousElementSibling in case whitespace appears?
https://www.w3schools.com/jsref/prop_node_previoussibling.asp

Reply:
change complete